test/cutting/test_cutting_roundtrip.py

```diff
@@ -179,3 +180,28 @@ def test_cutting_exact_reconstruction(example_circuit):
     logger.info("Max error: %f", np.max(np.abs(exact_expvals - simulated_expvals)))
 
     assert np.allclose(exact_expvals, simulated_expvals, atol=1e-8)
+
+
+def test_sampler_fail(example_circuit):
+    """This test checks if the sampler throws an error if you pass it a subscircuit with no measurements. Tests temporary workaround to Issue #422.
```

Comment:
```suggestion
def test_sampler_with_identity_subobservable(example_circuit):
    """This test ensures that the sampler does not throw an error if you pass it a subcircuit with no observable measurements.
    
    Tests temporary workaround to Issue #422.
```

I think this wording is more clear, as it focuses more on what is supposed to happen rather than the _incorrect_ behavior.